tts-evaluation-MCD-DTW.ipynb

```diff
@@ -0,0 +1,495 @@
+{
```

Comment:
You can just leave the audio samples that are used in the notebook (audio_0) and remove the rest